### PR TITLE
Sample Lightroom Partner App

### PR DESCRIPTION
**Pull request by slohia-adobe**

### CODE REVIEW

File: samples/partner_days_2021/views/index.jade [hunk 1]
```diff
@@ -0,0 +1,30 @@
+doctype
+html
+
+  head
+    title Adobe Lightroom Partner App Example
```
Comment (dkatyal) [inline samples/partner_days_2021/views/index.jade]: remove Adobe Lightroom, should be just Partner App Example

File: samples/partner_days_2021/views/index.jade [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    link(rel='stylesheet', href='/stylesheets/style.css')
+  body
+    header
+      h1 Adobe Lightroom Partner App Example
```
Comment (dkatyal) [inline samples/partner_days_2021/views/index.jade]: same as above

File: samples/src/common/lr/LrSession.mjs [hunk 3]
```diff
@@ -19,7 +19,9 @@ let _fetchAccountP = async (session) => {
 		return await LrRequestor.getP(session, '/v2/account')
 	}
 	catch (err) {
```
Comment (dkatyal) [inline samples/src/common/lr/LrSession.mjs]: is this file required?

File: samples/partner_days_2021/views/index.jade [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    .button-group
+      button(onclick="location.href='/login'") Log In with Adobe
+      button(onclick="location.href='/account'") Get Account
+      button(onclick="location.href='/catalog'") Get Catalog
+      button(onclick="location.href='/albums'") Get All Albums
+    br
+    br
+    form(action='/create_asset',method='post', enctype="multipart/form-data")
+               input(id='fileUpload',type='file',name='file')
+               button(type="submit") Create Image Asset
```
Comment (dkatyal) [inline samples/partner_days_2021/views/index.jade]: Should be Upload an image

File: samples/src/common/lr/LrSession.mjs (hunk 3 above)
Comment (slohia-adobe) [reply samples/src/common/lr/LrSession.mjs]: Removed

**Review by dkatyal — APPROVED** [review]
